packages/cached/src/cached-fs.ts

```diff
@@ -28,9 +36,13 @@ export interface IFailureCacheResult {
 
 export function createCachedFs(fs: IFileSystem): ICachedFileSystem {
   const getCanonicalPath = fs.caseSensitive ? identity : toLowerCase;
+  const contentCache = new Map<string, Buffer | string>();
   const statsCache = new Map<string, ISuccessCacheResult | IFailureCacheResult>();
   const { promises } = fs;
-  const invalidateAbsolute = (absolutePath: string) => statsCache.delete(getCanonicalPath(absolutePath));
+  const invalidateAbsolute = (absolutePath: string) => {
+    contentCache.delete(getCanonicalPath(absolutePath));
```

Comment:
`getCanonicalPath` once, and use result in both lines

Reply:
fixed